documentation/book/con-deploying-cluster-operator-using-operator-hub.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// assembly-cluster-operator.adoc
+
+[id='deploying-cluster-operator-using-operator-hub-{context}']
+= Deploying the Cluster Operator using OperatorHub.io
+
```

Comment:
_Please consider using the following text:_

{OperatorHub} is a catalog of Kubernetes Operators sourced from multiple providers. It offers you an alternative way to install stable versions of Strimzi using the Strimzi Kafka Operator.

The {OperatorLifecycleManager} is used for the installation and management of all Operators published on {OperatorHub}.

To install Strimzi from {OperatorHub}, locate the _Strimzi Kafka_ Operator and follow the instructions provided.

Reply:
Well, you rewrote everything :-o. But your text is better, so I used it.